src/containers/Locale/SelectLocale.jsx

```diff
@@ -6,43 +7,92 @@
  *
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import createHistory from 'history/createBrowserHistory';
 
-import { appLocales } from '../../i18n';
+import { appLocales, isValidLocale } from '../../i18n';
+
+class SelectLocale extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      locale: props.locale,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  async componentDidMount() {
+    const { locale } = this.state;
+    const { pathname } = window.location;
+
+    let storedLocale = localStorage.getItem('client_language');
+
+    // Only set nynorsk if browser is set to nynorsk, else default to nb
+    if (!storedLocale) {
+      if (locale === 'nn') {
+        storedLocale = locale;
+      } else {
+        storedLocale = 'nb';
+      }
+      localStorage.setItem('client_language', storedLocale);
```

Comment:
Jeg tror vi kun ønsker å lagre til localStorage ved at brukeren eksplisitt endrer språk. Hvis bruker kommer inn for første gang via `/en/etterellerannet` betyr ikke at han ønsker at språket skal settes til engelsk som default.